frontend/www/js/omegaup/components/course/TemporaryProblemList.vue

```diff
@@ -0,0 +1,219 @@
+                    v-model="problemAlias"
+                  ></omegaup-autocomplete
+                ></label>
+                <p class="help-block">
+                  {{ T.courseAddProblemsAssignmentsDesc }}
+                </p>
+              </div>
+              <div class="form-group col-md-4">
+                <label
+                  >{{ T.wordsPoints }}
+                  <input type="number" class="form-control" v-model="points" />
+                </label>
+              </div>
+            </div>
+            <div class="form-group text-right">
+              <button
+                data-add-problem
+                class="btn btn-primary mr-2"
+                type="submit"
+                v-bind:disabled="problemAlias.length == 0"
+                v-on:click.prevent="
+                  onAddProblem(assignment, {
+                    alias: problemAlias,
+                    points: points,
+                  })
+                "
+              >
+                {{ T.courseEditAddProblems }}
+              </button>
+            </div>
+          </div>
+        </div>
+      </form>
+    </div>
+    <!-- card-body -->
+  </div>
+  <!-- card -->
+</template>
+
+<style lang="scss" scoped>
+.omegaup-course-problemlist .form-group > label {
+  width: 100%;
+}
+
+.table td {
+  vertical-align: middle;
+}
+</style>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import { types } from '../../api_types';
+import T from '../../lang';
+import * as typeahead from '../../typeahead';
+import Autocomplete from '../Autocomplete.vue';
+
+import {
+  FontAwesomeIcon,
+  FontAwesomeLayers,
+  FontAwesomeLayersText,
+} from '@fortawesome/vue-fontawesome';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+import { library } from '@fortawesome/fontawesome-svg-core';
+library.add(fas);
+
+@Component({
+  components: {
+    'omegaup-autocomplete': Autocomplete,
+    'font-awesome-icon': FontAwesomeIcon,
+    'font-awesome-layers': FontAwesomeLayers,
+    'font-awesome-layers-text': FontAwesomeLayersText,
+  },
+})
+export default class CourseTemporaryProblemList extends Vue {
+  @Prop() assignments!: types.CourseAssignment[];
+  @Prop() assignmentProblems!: types.ProblemsetProblem[];
+  @Prop() taggedProblems!: omegaup.Problem[];
+  @Prop() selectedAssignment!: types.CourseAssignment;
+  @Prop() visibilityMode!: omegaup.VisibilityMode;
+
+  typeahead = typeahead;
+  T = T;
+  VisibilityMode = omegaup.VisibilityMode;
+  assignment: Partial<types.CourseAssignment> = this.selectedAssignment;
+  problems: types.AddedProblem[] = this.assignmentProblems;
+  showForm = omegaup.VisibilityMode.New;
+  difficulty = 'intro';
+  topics: string[] = [];
+  taggedProblemAlias = '';
+  problemAlias = '';
+  points = 100;
+  showTopicsAndDifficulty = false;
+
+  get tags(): string[] {
+    let t = this.topics.slice();
+    t.push(this.difficulty);
+    return t;
```

Comment:
nit: [`Array.concat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) hace lo que quieres:

```suggestion
    return this.topics.concat([this.difficulty]);
```

Reply:
Listo